pf4j/src/main/java/org/pf4j/PluginClassLoader.java

```diff
@@ -46,7 +47,7 @@
 
     private PluginManager pluginManager;
     private PluginDescriptor pluginDescriptor;
-    private boolean parentFirst;
+    private Set<ClassLoadingSource> classLoadingOrder;
```

Comment:
Here I see only a `ClassLoadingStrategy` instance that is an `Iterable` of `ClassLoadingStrategy.Source` (enum).

Reply:
done